### DISCUSSION TITLE
docker run hangs

### DISCUSSION THREAD

**Discussion by jmls** [kata-containers/kata-containers#6534] [Q&A] [answered]
I followed the installation steps on a brand-new ubuntu vm running under virtualbox

everything seemed to install ok - no errors, everything reports as it should

except that the final step, the docker test, fails to actually do anything

![image](https://user-images.githubusercontent.com/929523/228039373-5fe0b4e9-a97e-42b4-9b00-cfb498a1718a.png)

ctrl-c does nothing

docker ps shows nothing

If I go to another terminal and kill the process running /opt/kata/bin/containerd-shim-kata-v2 then the first terminal ends with 

"Docker: Error response from daemon: failed to create shim task: create container timeout: 74cc4a16774ddbd897394d792c2462b1020ce37998d7bfa952f6032cc9069677: unknown.
ERRO[0281] error waiting for container:      "

Something obviously isn't right - but probably my setup 

Just wondering if anyone else has hit this issue

**Comment by jmls** [comment]
ok, so I had the same problem on my ubuntu desktop. fixed it by removing vmware workstation, manually removing the vmware klernel modules, then imstalling the latest version of the kernel and rebooting ...

**Comment by GabyCT** [comment]
@jmls looking your screenshot seems that you are not running the container with `docker` but with `ctr`, that is why if you do `docker ps -a` you will see nothing as it seems that you are running the following command `sudo ctr run ...` to look for the container that you create with `ctr` you need to do `sudo ctr c list` or `sudo ctr t list`

**Reply by GabyCT** [reply]
Now if you want to run `kata` with `docker` you only need to install kata and try to run the following command for example (here KATA_HYPERVISOR can be qemu for example)
```
sudo docker run -ti --rm --runtime io.containerd.kata-${KATA_HYPERVISOR}.v2 $IMAGE bash
```
here $IMAGE is the image that you want to run and that needs to be a docker hub image

**Reply by GabyCT** [reply, marked as answer]
if you are looking how to install kata you can see this reference https://github.com/kata-containers/kata-containers/blob/main/docs/install/README.md
